doc/cephadm/services/index.rst

```diff
@@ -505,6 +505,57 @@ a spec like
 
 which would cause each mon daemon to be deployed with `--cpus=2`.
 
+Custom Config Files
+===================
+
+Cephadm supports specifying miscellaneous config files for daemons.
+To do so, users must provide both the content of the config file and the
+location within the daemon's container in which they want it mounted. After
+applying a YAML spec with custom config files specified and having cephadm
+redeploy the daemons for which the config files are for, these files will
+be mounted within the daemon's container at the specified location.
+
+The format within the YAML for doing this is
```

Comment:
within the service spec YAML file?

Reply:
changed it to just `Example service spec:`. Thoughts @anthonyeleven ?